docker/README.md

```diff
@@ -0,0 +1,84 @@
+# Install CAMEL with Docker
+
+Docker offers an easy way to create a consistent and isolated virtual
+environment, containers, for setting up the dependencies of CAMEL. This guide
+will show you how to quickly set up CAMEL, run the examples, and also
+develop on it, with Docker.
+
+## Prerequisites
+- Docker：https://docs.docker.com/engine/install/
+- Docker Compose：https://docs.docker.com/compose/install/
```

Comment:
maybe make it more clear that only one of the installation is required

Reply:
The installation of these two must be installed in order. In fact, the dependencies they require are independent of each other.